package.json

```diff
@@ -25,7 +25,9 @@
   },
   "dependencies": {
     "discord.js": "^11.5.1",
+    "fs": "0.0.1-security",
```

Comment:
This is almost certainly not the package you want.

You probs want fs, which is built into node directly.